tests/integration/test_ctgan.py

```diff
@@ -184,3 +184,14 @@ def test_wrong_sampling_conditions():
 
     with pytest.raises(ValueError):
         ctgan.sample(1, 'discrete', "d")
+
+
+def test_ctgan_data_transformer_params():
```

Comment:
I think you should also add a performance test, something simple just to make sure that our results are not worse than before because of this change.

Reply:
I'm not sure about this one, do you think about a performance test of the gaussian mixture model or CTGAN ? In terms of speed or accuracy ?

Reply:
Accuracy for CTGAN. Basically, just a test to make sure the changes don't break the code. So something like changing your `continuous` column to be a normal distribution, instead of random, then sample from the model (after you fit) and make sure the samples loosely follow a normal distribution.